modules/lvms-download-log-files-and-diagnostics.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_content-type: PROCEDURE
+[id="lvms-dowloading-log-files-and-diagnostics_{context}"]
+= Downloading log files and diagnostic information using must-gather
+
+When {lvms} is unable to automatically resolve a problem, use the must-gather tool to collect the log files and diagnostic information so that you or the Red Hat Support can review the problem and determine a solution.
+
+* Run the must-gather command from the client connected to {lvms} cluster by running the following command:
```

Comment:
I left a similar comment above, more of a nit, but the bullet is unnecessary for a single-step procedure.